crypto/src/hash/mod.rs

```diff
@@ -14,7 +14,10 @@ mod sha;
 pub use sha::Sha3_256;
 
 mod rescue;
-pub use rescue::{Rp62_248, Rp64_256};
+pub use rescue::{Rp62_248, Rp64_256, RpJive64_256};
+
+mod griffin;
+pub use griffin::GriffinJive64_256;
```

Comment:
I was also not sure about the final naming on the module and the struct.
Same goes for the instantiation of RP.